### PR TITLE
Add basic vertex property support for grpc

### PR DESCRIPTION
**Pull request by farodin91**
Part of #1853

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there an issue associated with this PR? Is it referenced in the commit message?
- [x] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [x] Have you written and/or updated unit tests to verify your changes?

### CODE REVIEW

File: janusgraph-grpc/src/main/proto/janusgraph/types/v1/schema_types.proto [hunk 1]
```diff
@@ -21,6 +21,54 @@ import "google/protobuf/wrappers.proto";
 option java_multiple_files = true;
 option java_package = "org.janusgraph.graphdb.grpc.types";
 
+enum PropertyDataType {
+    // java class: String.class
+    STRING = 0;
```
Comment (porunov) [inline janusgraph-grpc/src/main/proto/janusgraph/types/v1/schema_types.proto:26]: Usually by convention it is suggested to use `UNSPECIFIED` element for `0` enum value.
https://developers.google.com/protocol-buffers/docs/style#enums

Do you think we should follow this style convention? I don't have preference but thought could be good to discuss it

File: janusgraph-grpc/src/main/proto/janusgraph/types/v1/schema_types.proto [hunk 2]
```diff
@@ -21,6 +21,56 @@ import "google/protobuf/wrappers.proto";
...
+    google.protobuf.Int64Value id = 1;
+    // The VertexProperty Name.
+    string name = 2;
+    // The Data type.
+    PropertyDataType data_type = 3;
+    enum Cardinality {
+        CARDINALITY_UNSPECIFIED = 0;
+        // Vertex can have the property just once.
+        CARDINALITY_SINGLE = 1;
+        // Vertex can have the multiple property of the same.
```
Comment (FlorianHockmann) [inline janusgraph-grpc/src/main/proto/janusgraph/types/v1/schema_types.proto:65]:
```suggestion
        // Vertex can have multiple properties of the same name.
```

same below

File: janusgraph-grpc/src/main/proto/janusgraph/types/v1/schema_types.proto [hunk 3]
```diff
@@ -30,6 +80,8 @@ message VertexLabel {
     bool read_only = 3;
     // Marking as partitioned, default not partitioned.
     bool partitioned = 4;
+    // Contains all constrainted properties at the moment.
```
Comment (FlorianHockmann) [inline janusgraph-grpc/src/main/proto/janusgraph/types/v1/schema_types.proto:83]:
```suggestion
    // Contains all constrained properties at the moment.
```

**Review by FlorianHockmann — APPROVED** [review]
The changes look good to me. It's probably a good idea that we review the API again when we add some docs for them. I think that will make it easier to inspect how the API feels from a user's point of view

File: janusgraph-grpc/src/main/proto/janusgraph/types/v1/schema_types.proto (hunk 1 above)
Comment (farodin91) [reply janusgraph-grpc/src/main/proto/janusgraph/types/v1/schema_types.proto]: fixed

**Comment by farodin91** [comment]
@porunov Do you want to review it again?

File: janusgraph-grpc/src/main/proto/janusgraph/types/v1/schema_types.proto (hunk 2 above)
Comment (FlorianHockmann) [reply janusgraph-grpc/src/main/proto/janusgraph/types/v1/schema_types.proto]: Your updated commit fixed this comment here, but the comment two lines below is exactly like this.

**Pull request merged**